src/twisted/conch/scripts/ckeygen.py

```diff
@@ -189,7 +189,7 @@ def changePassPhrase(options):
             'Enter file in which the key is (%s): ' % filename)
     try:
         key = keys.Key.fromFile(options['filename'])
-    except keys.EncryptedKeyError as e:
+    except keys.EncryptedKeyError:
```

Comment:
This seems… extraneous, but I'll allow it :)

Reply:
failing txchecker because it hates me >:C